src/brokers/view-brokers/components/PreConfirmDeleteModal/PreConfirmDeleteModal.test.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  fireEvent,
+  render,
+  screen,
+  waitFor,
+  waitForI18n,
+} from '../../../../test-utils';
+import { PreConfirmDeleteModal } from '../PreConfirmDeleteModal/PreConfirmDeleteModal';
+jest.mock('react-i18next');
+
+describe('PreConfirmDeleteModal', () => {
+  const onDeleteButtonClick = jest.fn();
+  const onOpenModal = jest.fn();
+  const name = 'test-1';
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should modal renders correctly', async () => {
+    const comp = render(
+      <PreConfirmDeleteModal
+        onDeleteButtonClick={onDeleteButtonClick}
+        isModalOpen={true}
+        onOpenModal={onOpenModal}
+        name={name}
+      />,
+    );
+    await waitForI18n(comp);
+    expect(screen.getByText('delete_modal_instance_title')).toBeInTheDocument();
```

Comment:
Later we will use translation value instead of key `delete_modal_instance_title `. For now it's fine as we don't have setup for translation value.